src/frontend/src/components/NavBarV2/style.ts

```diff
@@ -0,0 +1,192 @@
+import styled from "@emotion/styled";
+
+export const AnnouncementBanner = styled.div`
+  background: #511cc1;
+  color: white;
+  text-align: center;
```

Comment:
Does this collection of font properties correspond to any of our existing fontBody styling classes?